### PR TITLE
Improve customization of outputs.

### PR DESCRIPTION
**Pull request by MurzynJ**
Allow to select if stdout backend and examples for any backend are built.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,9 @@ set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
 
 option(ENABLE_CONAN "use conan for consuming dependencies" on)
 option(ENABLE_TESTING "enable tests" off)
+option(NL_BUILD_EXAMPLES "build examples" on)
```
Comment (keba-uso) [inline CMakeLists.txt:11]: Please change the newly added option to _ENABLE_EXAMPLES_, to preserve the overall structure. Also, the examples should be disabled by default:
`option(ENABLE_EXAMPLES "enable examples" off)`

**Review by keba-uso — CHANGES_REQUESTED** [review]
Hi. I was assigned to review your open PRs within the nodesetLoader repository, so we could speed up the merge process.

I've discussed with @andreasebner, and from now on each new commit within the nodesetLoader should include the same commit convention as on the open62541 repository.

Please change commit title to be compatible with the commit convention, for example:

- build(pack): Improve customization of outputs

**Comment by keba-uso** [comment]
Also, please add the PR number in the commit message header: (#197)

File: CMakeLists.txt (hunk 1 above)
Comment (MurzynJ) [reply CMakeLists.txt:11]: Good point about the name. Will be changed.
Changing the default to "off" breaks the current behaviour, which is to always build the examples. But I guess this is what we want. I will change is as you suggested.

Comment (keba-uso) [reply CMakeLists.txt]: Makes sense to keep both stdout and examples ON. Nice!

**Pull request merged**